modular_skyrat/modules/modular_reagents/code/debuffs.dm

```diff
@@ -0,0 +1,51 @@
+/datum/status_effect/grouped/stasis_mobile
+	id = "stasis"
+	duration = -1
+	tick_interval = 10
+	alert_type = /atom/movable/screen/alert/status_effect/stasis
+	var/last_dead_time
```

Comment:
document this.

Reply:
I added comments on the file - what, exactly, are we looking for as far as documenting this goes?